src/TriggerBinding/SqlTriggerBindingProvider.cs

```diff
@@ -65,18 +66,25 @@ public Task<ITriggerBinding> TryCreateAsync(TriggerBindingProviderContext contex
                 return Task.FromResult<ITriggerBinding>(null);
             }
 
-            if (!IsValidTriggerParameterType(parameter.ParameterType))
+            Type parameterType = parameter.ParameterType;
+            if (!IsValidTriggerParameterType(parameterType))
             {
-                throw new InvalidOperationException($"Can't bind SqlTriggerAttribute to type {parameter.ParameterType}." +
-                    " Only IReadOnlyList<SqlChange<T>> is supported, where T is the type of user-defined POCO that" +
-                    " matches the schema of the user table");
+                throw new InvalidOperationException($"Can't bind SqlTriggerAttribute to type {parameter.ParameterType}.");
             }
 
             string connectionString = SqlBindingUtilities.GetConnectionString(attribute.ConnectionStringSetting, this._configuration);
 
-            // Extract the POCO type 'T' and use it to instantiate class 'SqlTriggerBinding<T>'.
-            Type userType = parameter.ParameterType.GetGenericArguments()[0].GetGenericArguments()[0];
-            Type bindingType = typeof(SqlTriggerBinding<>).MakeGenericType(userType);
+            Type bindingType;
+            if (parameterType == typeof(string))
+            {
+                bindingType = typeof(SqlTriggerBinding<>).MakeGenericType(typeof(JObject));
```

Comment:
Add comment explaining why parameter type of string maps to binding type of JObject